coala_langserver/langserver.py

```diff
@@ -2,77 +2,89 @@
 import argparse
 import socketserver
 import traceback
+import logging
 
-from .jsonrpc import JSONRPC2Connection, ReadWriter, TCPReadWriter
-from .log import log
```

Comment:
removing this (and other logging enhancements) should be done as a separate PR.  I've assigned https://github.com/coala/coala-vs-code/issues/36 to you, as you've already done a lot of this.

Reply:
Noted.